recipe-box-backend/src/main/java/com/endava/recipebox/response/CustomRecipe.java

```diff
@@ -0,0 +1,41 @@
+package com.endava.recipebox.response;
+
+import com.endava.recipebox.model.*;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.Duration;
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CustomRecipe  {
```

Comment:
A best practice in naming of Objects that we send on the response is to be named as RecipeDTO, DTO = Data Transfer Object.